react-with-json-logic/src/models/questionnaire.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Questionnaire as QuestionnaireBase,
+  AnyQuestion as AnyQuestionBase,
+  ResultCategory as ResultCategoryBase,
+  Result as ResultBase,
+  Variable as VariableBase,
+  QuestionnaireMeta as QuestionnaireMetaBase,
+  ISOLanguage,
+} from "covquestions-js/models/Questionnaire.generated";
+
+export interface Questionnaire extends QuestionnaireBase {
+  questions: AnyQuestion[];
+  resultCategories: ResultCategory[];
+  variables: Variable[];
+}
+
+export type AnyQuestion = AnyQuestionBase & {
+  enableWhenString?: string;
+};
+
+export interface ResultCategory extends ResultCategoryBase {
+  results: Result[];
+}
+
+export interface Result extends ResultBase {
+  valueString?: string; // TODO: Maybe we can come up with a better name than value.
```

Comment:
Should we rename them also? Maybe something like expressionString? Or logicString?

Reply:
Actually I wanted to rename it to `expressionString`.  But then I somehow got over it.